src/helpers/movie.helper.ts

```diff
@@ -128,7 +128,7 @@ export const getPoster = (el: HTMLElement): string => {
 export const getDescriptions = (el: HTMLElement): string[] => {
   // TODO more plots
   const plot = el
-    .querySelector('.body--plots .plot-preview p')
+    .querySelector('.body--plots .plot-full p')
```

Comment:
I have tried this query and it obtains descriptions for movies that have 1, 2, or 3 descriptions.
```js
let descriptions = []
document.querySelectorAll('.body--plots .plot-full p, .body--plots .plots .plots-item p').forEach((movie) => descriptions.push(movie.textContent?.trim().replace(/(\r\n|\n|\r|\t)/gm, '')))
```